app/assets/stylesheets/layout_components/notification.scss

```diff
@@ -111,3 +111,102 @@
     }
   }
 }
+/* Notification Centre Styling starts here */
+.right-sidebar{
+  z-index: 9999;
+  width: 350px;
+  transform: translate(350px, 0px);
+
+  &__bg{
```

Comment:
I don't know if you need this bg class - you could just add the background colour to .right-sidebar?